jobs/config.json

```diff
@@ -476,10 +476,11 @@
 "ci-kubernetes-e2e-kubeadm-gce": {
   "scenario": "kubernetes_e2e",
   "args": [
+    "--cluster=",
```

Comment:
emmm guess I'll keep your previous cluster name since it's not running from Jenkins, 1sec patching up